webpack/manifest.ts

```diff
@@ -62,7 +62,7 @@ const buildManifestV3 = (vendor: string): Manifest.WebExtensionManifest => {
 
     web_accessible_resources: [
       {
-        matches: ['https://*/*'],
+        matches: ['https://*/*', 'http://*/*'],
```

Comment:
Why do we need `http://` permission now?

Reply:
You may be wondered but some HTTP sites contain Google ads, which we haven't replaced before. Site example: http://stalivyrazy.org.ua/everyfile2.php?transfer=folders/ha/hrjn.html . By the way, this site appears at the top of Google search